### DISCUSSION TITLE
Pokémon: Let's Go, Pikachu! & Pokémon: Let's Go, Eevee!

### DISCUSSION THREAD

**Discussion by Thiago86** [skyline-emu/skyline#694] [Game Requests]
I would like to see this game  Pokémon lets Go Pikachu

**Comment by Nemomando** [comment]
Remente rodar ele  seri muito legal

**Comment by Ezekielsoyer** [comment]
J'ai téléchargé le jeu Pokémon let's go Pikachu NSP mais quand j'ouvre le jeu sur l'émulateur Skyline les touches apparaissent mais aucune image et aucun song au bout d'un moment Skyline crash qu'es ce que je dois faire

**Reply by Ibogl** [reply]
Je crois que c'est pas encore compatible ce jeu

**Comment by Pasa57** [comment]
Does any pokemon game run? 
I Tried lets go pikachu and sword. Boths doesnt run

**Reply by Mbarekk00** [reply]
How do I download the games?

**Reply by PokemonKing12** [reply]
From nswgames.com

**Comment by AleWiccan** [comment]
Use release1351 and apply the settings corresponding to that game (tested on version NSP 1.0.0)

**Reply by tmzy00** [reply]
Chinese Netizens Have Solved the Problem That the Current Version can't Open Pokémon Go Pikachu
